buildSrc/src/main/groovy/openhouse.iceberg-conventions-1.2.gradle

```diff
@@ -0,0 +1,11 @@
+ext {
+  icebergVersion = '1.2.0'
```

Comment:
Is this convention for backward compatibility reason?

Reply:
The plan is upgrade the server and client separately.